packages/image_picker/image_picker_windows/pubspec.yaml

```diff
@@ -17,7 +17,7 @@ flutter:
 
 dependencies:
   file_selector_platform_interface: ^2.2.0
-  file_selector_windows: ^0.8.2
+  file_selector_windows: ">=0.8.2 <0.10.0"
```

Comment:
why is the <0.10.0 needed?

Reply:
The current version is 0.9.x. I thought I would need to do a range to avoid conflicting with `file_selector`'s dependency, which I assumed was still on 0.8.0 since build-all hadn't been throwing conflicting dependency errors, but it turns out it's already on 0.9.x, so now I'm confused about why the existing 0.8.2 wasn't a problem.

But since file_selector is already on 0.9, I can just make this a simple update.